Source/WebCore/page/PointerLockController.cpp

```diff
@@ -40,44 +42,83 @@
 
 namespace WebCore {
 
+class UnadjustedMovementPlatformMouseEvent : public PlatformMouseEvent {
+public:
+    UnadjustedMovementPlatformMouseEvent(const PlatformMouseEvent &src)
+        : PlatformMouseEvent(src.position(), src.globalPosition(), src.button(), src.type(), src.clickCount(), src.modifiers(), src.timestamp(), src.force(), src.syntheticClickType(), src.pointerId())
+    {
+        m_movementDelta = m_unadjustedMovementDelta = src.unadjustedMovementDelta();
```

Comment:
Can you break this up into multiple statements?
https://webkit.org/code-style-guidelines/#linebreaking-chained-assignments

Reply:
@james-howard I think you missed this one